pkg/maps/lbmap/lbmap.go

```diff
@@ -225,12 +225,12 @@ func (*LBBPFMap) AddBackend(id uint16, ip net.IP, port uint16, ipv6 bool) error
 	}
 
 	if ipv6 {
-		backend, err = NewBackend6(loadbalancer.BackendID(id), ip, port, u8proto.ANY)
+		backend, err = NewBackend6V2(loadbalancer.BackendID(id), ip, port, u8proto.ANY)
 	} else {
-		backend, err = NewBackend4(loadbalancer.BackendID(id), ip, port, u8proto.ANY)
+		backend, err = NewBackend4V2(loadbalancer.BackendID(id), ip, port, u8proto.ANY)
 	}
 	if err != nil {
-		return fmt.Errorf("Unable to create backend (%d, %s, %d, %t): %s",
+		return fmt.Errorf("Unable to create backend v2 (%d, %s, %d, %t): %s",
```

Comment:
Nit: I would drop this change. There is a single `AddBackend` method, so if we see this error message, we already know where it comes from and what it refers to :-)

Reply:
Makes sense, removed :)